src/adapter/templates/getStringyProps.ts

```diff
@@ -15,7 +15,12 @@ export const getStringyProps = remoteFunction(function (this: unknown, maxLength
   }
 
   for (const [key, value] of Object.entries(this)) {
-    if (typeof value === 'object' && value && !String(value.toString).includes('[native code]')) {
+    if (
+      typeof value === 'object' &&
+      value &&
+      !String(value.toString).includes('[native code]') &&
+      !String(this).includes('[object ')
```

Comment:
```suggestion
      !String(value).includes('[object ')
```

(also, please apply the suggestion below to this one)